src/assets/wise5/themes/default/themeComponents/stepTools/step-tools.component.ts

```diff
@@ -43,14 +44,17 @@ export class StepToolsComponent implements OnInit {
   protected nodeStatus: any;
   protected nodeStatuses: any;
   protected prevId: string;
+  @Input() private timedStep: boolean;
```

Comment:
Instead of making timedStep an input to this component, can we set this in updateModel()?
```suggestion
  private timedStep: boolean;
```